components/eam/src/physics/crm/samomp/crm_module.F90

```diff
@@ -369,10 +369,10 @@ subroutine crm(lchnk, ncrms, dt_gl, plev,       &
   crm_rad_qi  = 0.
   crm_rad_cld = 0.
 #ifdef m2005
-  crm_rad%nc = 0.0
-  crm_rad%ni = 0.0
-  crm_rad%qs = 0.0
-  crm_rad%ns = 0.0
+  crm_rad_nc = 0.0
```

Comment:
These don’t look like they ever get associated now? I’m not seeing where these are associated to the crm_rad fields.

Reply:
It happens in the allocate routine.